stdlib/LinearAlgebra/test/blas.jl

```diff
@@ -241,6 +241,46 @@ Random.seed!(100)
             end
         end
 
+        # spmv!
+        if elty in (Float32, Float64)
+            @testset "spmv!" begin
+                # Both matrix dimensions n coincide, as we have symmetric matrices.
+                # Define the inputs and outputs of spmv!, y = α*A*x+β*y
+                α = rand(elty)
+                M = rand(elty, n, n)
+                A = (M+M')/elty(2.0)
+                x = rand(elty, n)
+                β = rand(elty)
+                y = rand(elty, n)
+
+                y_result_julia = α*A*x+β*y
+
+                # Create lower triangular packing of A
+                AP = typeof(A[1,1])[]
+                for j in 1:n
+                    for i in j:n
+                        push!(AP, A[i,j])
+                    end
+                end
```

Comment:
I'd suggest to make the test a bit "fancier". 😄  Have `A = Symmetric(M)`, and use `M` in the loop to build `AP`, so never really build a symmetric matrix in memory... just a suggestion.